xml/http_servers.xml

```diff
@@ -1689,13 +1689,21 @@
     <param pos="0" name="service.component.product" value="JBossWeb"/>
     <param pos="2" name="service.component.version"/>
   </fingerprint>
-  <fingerprint pattern="^Servlet/[\d\.]+; JBossAS-(.*)$">
+  <fingerprint pattern="^Servlet\/[\d\.]+; JBossAS-(.*)$">
     <description>JBoss AS</description>
     <example>Servlet/3.0; JBossAS-6</example>
```

Comment:
Since you are changing the regex, please complete the example and test the extraction.